imls-backend/db/schema.sql

```diff
@@ -9,6 +9,13 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
+--
+-- Name: admin; Type: SCHEMA; Schema: -; Owner: -
+--
+
+CREATE SCHEMA admin;
```

Comment:
I'm not sure this updated schema reflects the migration since I see stuff like `api.test` that are not in migrations. Also, I see a number of additional migrations in the schema even though we only have one migration here (such as 20220811192329 and 20220811194424 and 20220811195049). Can we do a `dbmate dump` after running this migration to clean up the schema?

Alternatively, we may have forgotten to run `dbmate dump` for a previous PR, in which case i think we should go ahead and do that and commit the resulting schema to main. What do you think @meenubhooshanan ?

Reply:
(the migration itself looks good though!)

Reply:
@cantsin , should we go ahead and merge, and do a cleanup as a separate ticket?

Reply:
I can clean up on this branch, commit&push, then merge to main? Just seems faster to me, but lmk if I'm missing something.

Reply:
Unfortunately, these orphaned migrations break for me because I can't do `dbmate down` or `dbmate up`. `dbmate` looks up these migrations and complains when they are not found.

Reply:
I think all we have to do here is to remove the migration `20220810132049`.

I wiped out my database, and did a clean `dbmate up` then `dbmate sync`. which resulted in this diff:

```diff
diff --git a/imls-backend/db/schema.sql b/imls-backend/db/schema.sql
index 958abf7..08f40a4 100644
--- a/imls-backend/db/schema.sql
+++ b/imls-backend/db/schema.sql
@@ -30,20 +30,6 @@ CREATE SCHEMA api;
 CREATE SCHEMA imlswifi;


---
--- Name: test(); Type: FUNCTION; Schema: api; Owner: -
---
-
-CREATE FUNCTION api.test() RETURNS TABLE(start_time timestamp with time zone, end_time timestamp with time zone)
-    LANGUAGE plpgsql
-    AS $$
-BEGIN
-RETURN QUERY
-SELECT presences.start_time, presences.end_time
-FROM api.presences;
-END; $$;
-
-
 SET default_tablespace = '';

 SET default_table_access_method = heap;
@@ -460,7 +446,6 @@ INSERT INTO public.schema_migrations (version) VALUES
     ('20220727163656'),
     ('20220729132839'),
     ('20220729150547'),
-    ('20220810132049'),
     ('20220811192329'),
     ('20220811194424'),
     ('20220811195049'),
```

Reply:
@cantsin I think I just fixed the schema file to correspond with the migrations... can you confirm when you get a chance?

Reply:
Yup! Think we both just caught it. Just committed the schema.sql w/out the 20220810 migration.

Reply:
Almost! I see the `api.test` is gone. But I still see `20220810132049` as a migration. Do you have that locally?